src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -62,10 +64,11 @@ class UserService(
   fun getUserForRequest(): UserEntity {
     val deliusPrincipal = httpAuthService.getDeliusPrincipalOrThrow()
     val username = deliusPrincipal.name
+    val serviceForRequest = requestContextService.getServiceForRequest()
```

Comment:
originally i threw an exception here if this couldn't be resolved, but 500+ integration tests broke because they're not setting the service name header, so for now i've allowed it to be nullable

Reply:
Resolving this as i've added this behaviour back in, i'm looking at fixing tests